python.md

```diff
@@ -1233,6 +1234,18 @@ Related reading:
 
 - http://stackoverflow.com/questions/19074745/python-docstrings-descriptions-vs-comments
 
+### <a name="spelling">Spelling</a>
+
+Consistent spelling is helpful because it decreases the risk of incorrect variable references. Where applicable, follow the spelling used in external libraries. For example, the Django Rest Framework uses the American English spelling for `serializer` which is therefore prefered. Elsewhere aim for conformity with the pre-existing choices. For example, given the following split:
```

Comment:
Should be 'preferred' (ironic, eh?) ;)

Reply:
haha 🙄 inevitable. I'm so glad my witty irony isn't lost on everyone.

Reply:
I would go with something like:

> ### Prefer American English for naming modules and objects
> 
> When naming objects like modules, classes, functions and variables, prefer American English. Eg, use `serializers.py` instead of `serialisers.py`. This ensures the names in our codebase match those in the wider ecosystem.
> 
> UK spellings are fine for in comments or docstrings.